### ISSUE TITLE
HTML/Js/Css minification vs ServerEvents

### ISSUE THREAD

**Issue by aleksandrov** [ServiceStack/Issues#461]
Hello,

I have SS 4.0.60 self-hosted with ServerEventsFeature enabled (default parameters). Everything works just fine until I started to minify static files with in-memory VFS (using https://github.com/ServiceStack/ServiceStack/wiki/HTML,-CSS-and-JavaScript-Minification as a manual).

It is strange, but Server Events stops working, I got 404.

> Handler for Request not found (404):
> Request.HttpMethod: GET
> Request.PathInfo: /event-stream/
> Request.QueryString: 
> Request.RawUrl: /event-stream/

Please find below my GetVirtualFileSources(), I removed all stuff from article but it still does not work.

```csharp   
public override List<IVirtualPathProvider> GetVirtualFileSources()
{
    var existingProviders = base.GetVirtualFileSources();
    var memFs = new InMemoryVirtualPathProvider(this);
    existingProviders.Insert(0, memFs);
    return existingProviders;
}
```

I hope it should be possible to mix both features in one place!

**Comment by xplicit** [comment]
Can you provide a standalone sample code which shows the issue?

**Comment by aleksandrov** [comment]
Here is sample.

When I go to http://localhost:3000/event-stream?channels=test it redirects to http://localhost:3000/event-stream/ with 404.

When I comment out line "existingProviders.Insert(0, memFs)" it loads stream correctly.

```csharp
using Funq;
using ServiceStack;
using ServiceStack.IO;
using ServiceStack.VirtualPath;

namespace ConsoleApplication
{
    class Program
    {
        static void Main(string[] args)
        {
            var appHost = new AppHost().Init().Start("http://*:3000/");
            Console.ReadKey();
        }
    }

    class AppHost : AppSelfHostBase
    {
        public AppHost() : base("Sample", typeof(AppHost).Assembly)
        {
        }

        public override void Configure(Container container)
        {
            Plugins.Add(new ServerEventsFeature());
        }

        public override List<IVirtualPathProvider> GetVirtualFileSources()
        {
            var existingProviders = base.GetVirtualFileSources();
            var memFs = new InMemoryVirtualPathProvider(this);
            existingProviders.Insert(0, memFs);
            return existingProviders;
        }
    }
}
```

**Comment by mythz** [comment]
This should now be resolved from this commit: https://github.com/ServiceStack/ServiceStack/commit/d24b2d16cc28db43f2eb5d87afac54ca55904ed9

Which is available from v4.0.61 that's now [available on MyGet](https://github.com/ServiceStack/ServiceStack/wiki/MyGet).

thx for the repro!
